package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "sailthru-client",
     "description": "Node.js client for Sailthru API",
-    "version": "1.0.8",
+    "version": "2.0.0",
     "author": {
         "name": "Prajwal Tuladhar",
```

Comment:
Ah, we should remove them. We can put one of you guys as author instead. Or, myself or @ibwhite if you so prefer.